index.bs

```diff
@@ -839,6 +850,28 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  If <code>|credentialCreationData|.[=AttestationConveyancePreferenceOption=]</code>'s value is
+                    <dl class="switch">
+                        :   "none"
+                        ::  replace potentially uniquely identifying information (such as [=AAGUID=] and
+                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
+                            respectively, with blinded versions of the same data.
+
+                        Issue: need to define "blinding".  See issues [#694](https://github.com/w3c/webauthn/issues/694) and
```

Comment:
Bikeshed lets you write `Issue(694):` to link to the issue: https://tabatkins.github.io/bikeshed/#remote-issues

Reply:
its actually of the form "Issue(w3c/webauthn#694): ...issue text..."